packages/commuter-client/src/reducers/index.js

```diff
@@ -7,10 +7,16 @@ const commuter = (state = {}, action) => {
       return Object.assign({}, state, { isFetching: action.isFetching });
     case types.RECEIVE_CONTENTS:
     // filter out hidden files
-    const content = action.entry.type === 'directory' ?
-    action.entry.content.filter((file) => {
-      return !file.name.startsWith('.')
-    }) : action.entry.content;
+      let content;
+      switch (action.entry.type) {
+        case 'directory':
+          content = action.entry.content.filter((file) => {
```

Comment:
At least for the local backend, we'll be able to support that out of the box using the git utilities. :smile: